diff --git a/lib/http/parse.myr b/lib/http/parse.myr
index 8b14af6d..85862a1d 100644
--- a/lib/http/parse.myr
+++ b/lib/http/parse.myr
@@ -4,10 +4,10 @@ use bio
 use "types"
 
 pkg http =
-	pkglocal const parsereq		: (s : session# -> std.result(req#, err))
-	pkglocal const parseresp	: (s : session#, r : resp# -> bool)
-	pkglocal const parsechunksz	: (s : session# -> std.result(std.size, err))
-	pkglocal const parsenumber	: (str : byte[:]#, base : int -> std.option(int))
+	const parsereq		: (s : session# -> std.result(req#, err))
+	const parseresp	: (s : session#, r : resp# -> bool)
+	const parsechunksz	: (s : session# -> std.result(std.size, err))
+	const parsenumber	: (str : byte[:]#, base : int -> std.option(int))
 ;;
 
 const strcaseeq = {a : byte[:], b : byte[:] -> bool
diff --git a/lib/http/session.myr b/lib/http/session.myr
index 40911651..b77a064e 100644
--- a/lib/http/session.myr
+++ b/lib/http/session.myr
@@ -8,9 +8,9 @@ pkg http =
 	const mksrvsession	: (fd	: std.fd -> session#)
 	const freesession	: (s : session# -> void)
 
-	pkglocal const ioput	: (s : session#, fmt : byte[:], args : ... -> bool)
-	pkglocal const iowrite	: (s : session#, buf : byte[:] -> bool)
-	pkglocal const ioflush	: (s : session# -> void)
+	const ioput	: (s : session#, fmt : byte[:], args : ... -> bool)
+	const iowrite	: (s : session#, buf : byte[:] -> bool)
+	const ioflush	: (s : session# -> void)
 ;;
 
 const mksession = {schema, host, port
